### ISSUE TITLE
UFAL/PY-license_resource_mapping old 107679 != new 5023

### ISSUE THREAD

**Issue by milanmajchrak** [dataquest-dev/dspace-python-api#84]

**Comment by milanmajchrak** [comment]
1. I've checked count of bitstreams which has assigned license in CLARIN-DSpace5 database and that count is 5023 what is the same count than the count after the import. I think this is not a problem anymore.

**Comment by milanmajchrak** [comment]
2. The `license_resource_mapping` has multiple records for some bitstreams e.g. ,
![image](https://github.com/dataquest-dev/dspace-python-api/assets/90026355/e5516519-2844-4480-86b3-ec87eac7fbb0)
the mapping between `bitstream_id` and `license_id` is the same, but this record was added multiple times.

When I counted all records for each bitstream the final count was the 107679 what is the count of records from this table.
Select: 
```
select bitstream_id, count(*) from license_resource_mapping
	group by bitstream_id
	order by count(*) desc ;
```
then I sum `count(*)` column.
